pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -181,6 +181,7 @@ metadata:
   annotations:
     tkg.tanzu.vmware.com/add-missing-fields-from-tkr: #@ data.values.KUBERNETES_RELEASE
 spec:
+  #@ if data.values.CNI != "antrea" or antrea_configs_exist():
```

Comment:
We'll need to revisit this. I don't think the antrea_configs_exist() method is accurate. this can be a separate PR. we'll need to file and issue for it.